src/include/session.h

```diff
@@ -196,6 +203,8 @@ struct __wt_session_impl {
     char unittest_assert_msg[WT_SESSION_UNITTEST_BUF_LEN];
 #endif
 
+    WT_SESSION_DIAGNOSTIC_ASSERTS_LEVEL diagnostic_asserts_level;
```

Comment:
Can we add a comment here?

Reply:
Yep! Added in [e8c11aa](https://github.com/wiredtiger/wiredtiger/pull/8485/commits/e8c11aa41919ef9959b3d5e572d10f6d965a18b4)